docs/docs/project-configuration-dev-test-build.mdx

```diff
@@ -104,40 +104,28 @@ You can find all the details in the [source](https://github.com/redwoodjs/redwoo
 
 You can customize the types that Redwood generates from your project too! This is documented in a bit more detail in the [Generated Types](typescript/generated-types#customising-codegen-config) doc.
 
-## Debugger configuration
-The `yarn rw dev` command is configured by default to launch a debugger on the port `18911`, your Redwood app also ships with default configuration to attach a debugger from VSCode.
-
-Simply run your dev server, then attach the debugger from the "run and debug" panel. Quick demo below:
-
-<ReactPlayer width='100%' height='100%' controls url='https://user-images.githubusercontent.com/1521877/159887978-95075ccd-d10c-403c-90cc-a5b944c429e3.mp4' />
+## Debug configurations
 
+### Dev Server
+The `yarn rw dev` command is configured by default to open a browser and a debugger on the port `18911` and your redwood app ships with several default configurations to debug with VSCode.
 
-<br/>
-
-> **ℹ️ Tip: Can't see the "Attach debugger" configuration?** In VSCode
->
-> You can grab the latest launch.json from the Redwood template [here](https://github.com/redwoodjs/redwood/blob/main/packages/create-redwood-app/templates/ts/.vscode/launch.json). Copy the contents into your project's `.vscode/launch.json`
-
-
-#### Customizing the debug port
-You can choose to use a different debug port in one of two ways:
-
+#### Customizing the configuration
 **a) Using the redwood.toml**
 
-Add/change the `debugPort` under your api settings
+Add/change the `debugPort` or `open` under your api settings
 
 ```toml title="redwood.toml"
 [web]
   # .
-  # .
 [api]
-  port = 8911
+  # .
   // highlight-next-line
   debugPort = 18911 # 👈 change me!
+[browser]
+  // highlight-next-line
+  open = true # 👈 change me!
```

Comment:
I think highlighting is enough, you can remove the 👈  emoji. (And please do the same for the already existing one above)

Reply:
Done ✅